components/automate-ui/src/app/pages/service-groups/service-groups.component.ts

```diff
@@ -211,40 +210,34 @@ export class ServiceGroupsComponent implements OnInit, OnDestroy {
       withLatestFrom(this.route.queryParamMap),
       takeUntil(this.isDestroyed)
     )
-    .subscribe(([serviceGroups, queryParams]) => {
+    .subscribe(([serviceGroups]) => {
       if (serviceGroups.length > 0) {
-        // if we do not have an sgId parameter, pick the first sgId from the
-        // service group list and load it:
-        const sgId = queryParams.get('sgId');
-        if ( !sgId ) {
-          this.router.navigate(
-            [],
-            { queryParams: { sgId: serviceGroups[0]['id'] }, queryParamsHandling: 'merge' }
-          );
+        // if we do not have a selected service group, pick the first sgId from
+        // the service group list
+        if ( !this.selectedServiceGroupId ) {
+          this.selectedServiceGroupId = serviceGroups[0]['id'];
```

Comment:
In its existing form, you can replace `'id'` with anything and get no complaint. But if you change to this, you can only put valid field names:
```suggestion
          this.selectedServiceGroupId = serviceGroups[0].id;
```